### PR TITLE
replace prose text with bulleted algorithm for "How it works"

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

**Comment by adamralph** [comment]
@bording what do you think about this change? Is it easier to grok than the prose text?

File: README.md [hunk 1]
```diff
@@ -42,13 +42,21 @@ _NOTE: The MinVer package reference should normally include `PrivateAssets="All"
...
-If no tag is found either on the current commit or in the commit history, the default version `0.0.0-alpha.0` is used.
-
-You will notice that MinVer adds another number to the pre-release identifiers when the current commit is not tagged. This is the number of commits since the latest tag, or if no tag was found, since the root commit. This is known as "height". For example, if the latest tag found is `1.0.0-beta.1`, at a height of 42 commits, the calculated version is `1.0.0-beta.1.42`.
+- If the current commit is tagged with a version:
+  - The version is used as-is.
+- If the current commit is _not_ tagged with a version:
+  - MinVer searches the commit history for the latest commit tagged with a version.
+    - If a tag is found:
+      - If the version is a [pre-release](https://semver.org/spec/v2.0.0.html#spec-item-9):
+        - The version is used as-is.
```
Comment (bartelink) [inline README.md:51]: Need something to indicate that it is used as the baseline, e.g.:
"the tag is used as the basis (see 'height' below)"

File: README.md [hunk 2]
```diff
@@ -42,13 +42,21 @@ _NOTE: The MinVer package reference should normally include `PrivateAssets="All"
...
+      - If the version is RTM (not pre-release):
+        - The patch number is incremented, but this [can be customised](#can-i-auto-increment-the-minor-or-major-version-after-an-rtm-tag-instead-of-the-patch-version).
+        - Default pre-release identifiers are added. The default pre-release phase is "alpha", but this [can be customised](#can-i-change-the-default-pre-release-phase-from-alpha-to-something-else).
+        - For example, If the latest tag is `1.0.0`, the current version will be `1.0.1-alpha.0`.
+    - If no tag is found:
+      - The default version `0.0.0-alpha.0` is used.
```
Comment (bartelink) [inline README.md:57]: as the base and/or some other term that aligns with previous

**Comment by adamralph** [comment]
@bartelink @bording thanks for your comments.

I pushed another commit to address @bartelink's comments, and to improve the language a little further.

**Comment by adamralph** [comment]
I forgot to squash the fixup before merging. 🤦

**Comment by adamralph** [comment]
And for that reason, I've just installed https://github.com/apps/dontmergemeyet for my account. Thanks @thomaslevesque.

**Comment by thomaslevesque** [comment]
@adamralph you're welcome, glad you like it!

**Pull request merged**